src/System.Private.ServiceModel/tests/Stress/SharedPoolsOfWCFObjects/CallStats.cs

```diff
@@ -68,11 +64,15 @@ public T CallFuncAndRecordStats<T>(Func<T> func)
             catch (Exception e)
```

Comment:
NIT: I don't see `e` used 
(Question: are you building with Warnings as Errors?)